docs/reference.rst

```diff
@@ -586,6 +586,8 @@ You can use `openssl asn1parse` for retrieving the message digest in the CMS.
 .. note::
   The `asn1parse` method can also be used for the `verity` bundle
 
+`rauc replace-signature` can replace the bundle signature with a new one.
```

Comment:
Could you point to upstream documentation how one would create the new signature (perhaps links to the necessary OpenSSL functions)? 

This could also use a short 'pseudo-shell' example. The command to extract the signature, then refer to the custom creation and finally the cal to `rauc replace-signature`.

Reply:
Yes for sure